common/src/main/java/com/wynntils/core/features/Feature.java

```diff
@@ -173,7 +171,7 @@ public final void disable() {
 
         enabled = false;
 
-        ManagerRegistry.removeAllFeatureDependency(this);
+        ManagerRegistry.removeAllDependency(this);
```

Comment:
The grammar does not work out. Please rename to `removeAllDependencies` (I know this is a pre-existing problem, but since you're touching it anyway...)